src/a-runtime-console/kubernetes/ui/pipeline/input-action-dialog/input-action-dialog.component.ts

```diff
@@ -84,4 +88,76 @@ export class InputActionDialog {
   close() {
     this.modal.close();
   }
+
+  checkIfJenkinsIsUpAndProceedURL() {
+    let notification;
+    this.callJenkins()
+      .subscribe((response) => {
+        switch (response.status) {
+          case 200:
+            notification = {
+              message: `Got ${response.status}, Jenkins is up and running.`,
+              type: NotificationType.SUCCESS
+            };
+            this.notifications.message(notification as Notification);
+            this.invokeUrl(this.inputAction.proceedUrl);
+            return;
+          case 307:
+          case 302:
+            notification = {
+              message: `Got ${response.status}, Connecting to Jenkins.`,
+              type: NotificationType.INFO
+            };
+            break;
+          case 503:
+            notification = {
+              message: `Got ${response.status}, Cluster resources capacity is full. Waiting along.`,
+              type: NotificationType.INFO
+            };
+            break;
+          case 504:
+            notification = {
+              message: `Got ${response.status}, Request timed out. Trying again.`,
+              type: NotificationType.WARNING
+            };
+            break;
+          case 202:
+            notification = {
+              message: `Got ${response.status}, Jenkins is currently idled. Please wait while we start it. waiting along.`,
+              type: NotificationType.WARNING
+            };
+            break;
+          default:
+            notification = {
+              message: `Got status ${response.status}`,
+              type: NotificationType.DANGER
+            };
+            this.notifications.message(notification as Notification);
+            return;
+        }
+        this.notifications.message(notification as Notification);
+        setTimeout (this.checkIfJenkinsIsUpAndProceedURL(), 10000);
+      });
+    this.close();
+  }
+
+  callJenkins(): Observable<Response> {
+    const token_json = {
+      access_token: this.authService.getToken(),
+      token_type: 'Bearer'
+    };
+    let url = this.build.jenkinsBuildURL + '?token_json=' + encodeURIComponent(JSON.stringify(token_json));
+    return this.http
+      .get(url)
+      .map((response: Response) => {
+        return response;
+      })
+      .catch((error) => {
+        this.notifications.message({
+          message: 'There seems to be a problem with Jenkins',
```

Comment:
In a local instance connected to production, a redirect still occurs that results in this error being hit and unidling behavior not occurring. 

![pipelines-redirect](https://user-images.githubusercontent.com/5430520/39783944-bcf49560-52e4-11e8-9cf3-24389ab18160.png)

The only way to fix this is to configure the server to send the right response header, but that's not an option. It may work in production though.

It looks like the jenkins proxy is configured s.t.
1. If a request to jenkins.openshift.io/*, redirect to auth.
2. If redirected to auth, redirect back to jenkins.openshift.io/* with ?token set
3. If a request to jenkins.openshift.io/* with ?token set, redirect to jenkins.openshift.io/*

It would be nice if there was an endpoint that didn't involve redirecting as long as the auth header was set with the correct token. Maybe the entire redirect mechanism can be looked at too, it's a bit long and browsers will show 'Too many redirects' even though it eventually goes to the right place. I digress...

Reply:
Also something like: s/There seems to be a problem with Jenkins/There is a problem accessing Jenkins./

It's best to avoid 'probably' messages ('seems' in this case) for the user. It's not as polished.